src/app/handlers/new_member_role.handler.ts

```diff
@@ -33,9 +35,24 @@ export class NewMemberRoleHandler implements IHandler {
       return;
     }
 
-    member.addRole(<Role>this._roleCache[this._UNVERIFIED_ROLE]);
-    this.container.messageService.sendBotReport(
-      `User \`${member.user.tag}\` has been automatically unverified`
-    );
+    member.addRole(<Role>this._roleCache[this._UNVERIFIED_ROLE]).then(() => {
+      this._pingUserInVerify(member);
+      this.container.messageService.sendBotReport(
+        `User \`${member.user.tag}\` has been automatically unverified`
+      );
+    });
+  }
+
+  private _pingUserInVerify(member: GuildMember) {
+    if (!this._VERIFY_CHANNEL) {
+      this._VERIFY_CHANNEL = this.container.guildService
+        .get()
+        .channels.filter((chan: GuildChannel) => chan.name === Constants.Channels.Bot.Verify)
+        .first() as TextChannel;
+    }
+
+    this._VERIFY_CHANNEL.send(member.user.toString()).then((sentMsg) => {
+      (sentMsg as Message).delete(1000 * 10); //Delete after 10 seconds
```

Comment:
might not be long enough; make it 12h

Reply:
It doesn't block, but depending on how many people join at once, there could be a good few pings at once in the channel. Is that ok?